components/BottomNavigation/README.md

```diff
@@ -181,3 +186,65 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+ `-accessibilityLabel` The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
+
+`-accessibilityValue`  Set an appropriate
+[`accessibilityValue`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue)
```

Comment:
Nit: I think we can drop the link here. I don't think we're generally linking to Apple's developer docs.  I've also seen too many dead links to Apple's developers docs to believe they will survive more than a year. :-\

Reply:
Done